core/templates/components/skiplinks/skiplinks.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skipped_to_title": "Main Content"
```

Comment:
I would do:
```
{
  target_id: "main-content",
  target_text: "Main Content",
  link_text: "Skip to main content
}
```
and make the corresponding changes in the `.twig` file.

Reply:
Added!

Reply:
Thank you!